manage.sh

```diff
@@ -202,13 +202,13 @@ docker_build() {
     fi
 
     # define the docker image name
-    # /!\ HACK to get the user name /!\
-    GITHUB_USER=$(git remote get-url origin | sed 's/.*github\.com\/\([^\/]*\).*/\1/')
+    GITHUB_USER=$(echo "${GIT_URL}" | sed 's/.*github\.com\/\([^\/]*\).*/\1/')
     SEARX_IMAGE_NAME="${GITHUB_USER:-searx}/searx"
 
     # build Docker image
     echo "Building image ${SEARX_IMAGE_NAME}:${SEARX_GIT_VERSION}"
     sudo docker build \
+         --build-arg GIT_URL="${GIT_URL}" \
```

Comment:
Few lines below, I forgot to mention 
```
--build-arg LABEL_VCS_URL=$(git remote get-url origin)
```
-->
```
--build-arg LABEL_VCS_URL="${GIT_URL}"
```

But this change can LABEL_VCS_REF inconsistent with LABEL_VCS_URL.

Reply:
sorry .. I should have seen it .. i will fix it